### DISCUSSION TITLE
Scaling font's width or height separately

### DISCUSSION THREAD

**Discussion by irol** [QuestPDF/QuestPDF#522] [Ideas]
I noticed that I can only change size of the font. I would like change it's height or width separately.

**Comment by flew2bits** [comment]
You can use ScaleHorizontal or ScaleVertical to accomplish this with the caveat that it can only be applied to entire text block. That is, you can't have a wider span of text in another block.

For example:
```
page.Content().Column(column => column.Item().ScaleHorizontal(1.2f).Text(Placeholders.Paragraph());
```

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
